packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -543,6 +578,7 @@ export function AvailabilitySettings({
                 )}
               </div>
             </div>
+            <TimeBlocks control={form.control} />
```

Comment:
Can we show this conditionally if the user has a calendar connected?

Reply:
Done.